manager/api/jpa/src/main/java/io/apiman/manager/api/jpa/JpaStorage.java

```diff
@@ -347,6 +325,14 @@ public void updatePlugin(PluginBean pluginBean) throws StorageException {
         super.update(pluginBean);
     }
 
+    /**
+     * @see io.apiman.manager.api.core.IStorage#updateDeveloper(DeveloperBean)
+     */
+    @Override
+    public void updateDeveloper(DeveloperBean developer) throws StorageException {
+        // TODO: Implement
```

Comment:
@EricWittmann You can find it here. As we use only elasticsearch we had not the time to implement the other storage layer as well. For the quickstart, we must implement this.